packages/grid/_modules_/grid/components/column-headers.tsx

```diff
@@ -99,7 +99,7 @@ export const ColumnsHeader = React.memo(
           ref={ref}
           className={wrapperCssClasses}
           aria-rowindex={1}
-          tabIndex={0}
+          tabIndex={-1}
```

Comment:
```suggestion
```

Reply:
Unfortunately, it won't work. The linter fails without it :/

Reply:
The other solution is to remove the `role="row"`. I think it is good to have the linter complain about accessibility problems because these are easily neglected :D. In terms of if it should be focusable or not - `tabIndex={-1}` makes it not focusable.

Reply:
Why not silence the linter?

Reply:
I don't think that removing the role is an option. I think that we should silence the linter. We have seen many cases where the linter is wrong in the main repository. It goes what he cans, it's up to us to look at why. Reading at https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/interactive-supports-focus.md#rule-details, I think that the heuristic doesn't apply in our case, this is not part of a composite widget

With `tabIndex={-1}` the element is focusable, it's only not tabbable.

Reply:
I couldn't find reference to it in https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html

Reply:
Ok, let's silence it, how can I stop it from complaining?

Reply:
tabIndex={-1} is fine as there is a listener on tab that will change it

Reply:
Column headers should be tabbable/focusable if there are buttons in them

Reply:
We do roving tabIndex hence -1

Reply:
Can we resolve and merge this PR @oliviertassinari ?

Reply:
I don't understand: the only button I can see is:

<img width="389" alt="Capture d’écran 2020-10-20 à 18 03 53" src="https://user-images.githubusercontent.com/3165635/96612915-a25e1500-12fe-11eb-8550-cc04552fe9ad.png">

it can be focused on its own, not sure that it should be.

Reply:
> Column headers should be tabbable/focusable if there are buttons in them

Which won't be broken if we do:

```suggestion
```

IMHO, we should, by default, not add attributes, unless we know exactly what we are doing.

Reply:
Could you share a user journey that would leverage the extra tabIndex={-1} if we don't have one, I think that we should remoe it.

Reply:
cells already have a tab index -1

<img width="162" alt="Capture d’écran 2020-10-20 à 18 14 59" src="https://user-images.githubusercontent.com/3165635/96614253-354b7f00-1300-11eb-8f64-84b877ebb041.png">

Reply:
I'm ok with removing it but I don't know how to edit the lint configuration. The focusing was working without it before I added it. The only reason why it's there is because of the linter.

Reply:
If you are using VSCode eslint plugin, it should give you suggestions to fix it, otherwise you can use an example from the core https://github.com/mui-org/material-ui/blob/3d949c1617d9c4a873476173d2cd428136be809e/packages/material-ui-lab/src/TreeItem/TreeItemContent.js#L52

Reply:
Noted I thought there might be a better way but I've updated the PR.